aruco_opencv/src/aruco_tracker.cpp

```diff
@@ -555,7 +573,10 @@ class ArucoTracker : public rclcpp_lifecycle::LifecycleNode
     detection_pub_->publish(detection);
 
     if (debug_pub_->get_subscription_count() > 0) {
-      auto debug_cv_ptr = cv_bridge::toCvCopy(img_msg, "bgr8");
+      auto debug_cv_ptr = std::make_shared<cv_bridge::CvImage>();
+      debug_cv_ptr->header = cv_ptr->header;
+      debug_cv_ptr->encoding = cv_ptr->encoding;
+      debug_cv_ptr->image = cv_ptr->image.clone();  // copy elision
```

Comment:
Probably a simple copy constructor or copyTo would suffice, but all of those should be semantically identical.

Reply:
After reading further into the documentation, cv::Mat(const cv::Mat &) does not copy the underlying buffer. This code looks fine despite the totally unnecessary and wrong comment.